README.md

```diff
@@ -127,24 +118,85 @@ queues:
   - [low_priority, 1]
 ```
 
-The ```aws``` section is used to configure both the Aws objects used by Shoryuken internally, and also to set up some Shoryuken-specific config. The Shoryuken-specific keys are listed below, and you can expect any other key defined in that block to be passed on untouched to ```Aws::SQS::Client#initialize```:
+And setup ```aws``` options to use ```configure_client``` in `config/initializers/shoryuken.rb`:
 
-- ```account_id``` is used when generating SNS ARNs
-- ```sns_endpoint``` can be used to explicitly override the SNS endpoint
-- ```sqs_endpoint``` can be used to explicitly override the SQS endpoint
-- ```receive_message``` can be used to define the options passed to the http://docs.aws.amazon.com/sdkforruby/api/Aws/SQS/Client.html#receive_message-instance_method
+```ruby
+Shoryuken.configure_client do |config|
+  config.aws = {
+    sqs_endpoint:      ...,         # or ENV["AWS_SQS_ENDPOINT"]
+    sns_endpoint:      ...,         # or ENV["AWS_SNS_ENDPOINT"]
+    secret_access_key: ...,         # or ENV["AWS_SECRET_ACCESS_KEY"]
+    access_key_id:     ...,         # or ENV["AWS_ACCESS_KEY_ID"]
+    region:            "us-east-1", # or ENV["AWS_REGION"]
+    receive_message: {              # See http://docs.aws.amazon.com/sdkforruby/api/Aws/SQS/Client.html#receive_message-instance_method
```

Comment:
@h3poteto thanks for the update! 🍻 

I think we could remove the `sqs_endpoint` and `sns_endpoint` from the example above, it can mislead people making the believe it's required, usually they just want to copy&paste and try to fill that information, I guess most don't know about these endpoints, WDYT?